frontend/src/App.js

```diff
@@ -26,30 +27,46 @@ export default function App() {
 
     return (
         <Router>
-            <PageLayout>
-            <AppHeader />
             <Switch>
-              <Route exact path="/">
-                  <p> This is the start page</p>
-              </Route>
-              <Route exact path="/gymnast/:username">
-                  <GymnastPage watchlistDifficulty={watchlistDifficulty}/>
-              </Route>
-              <Route exact path="/difficulty">
-                  <GroupPage/>
-              </Route >
-                <Route exact path="/difficulty/:groupId/:familyId/:categoryId">
-                    <ListOfDifficultyItemPage onWatchlistDifficultyClick={onWatchlistDifficultyClick} isOnWatchlistDifficulty={isOnWatchlistDifficulty}/>
+                <Route exact path="/welcome">
+                    <WelcomePage/>
                 </Route>
```

Comment:
```suggestion
                <Route exact path="/welcome" component={WelcomePage}/>
```

What do you think about this ? Got this notation from Jan ✌🏻